### PR TITLE
Add initial security document for running PTFE

### PR DESCRIPTION
**Pull request by evanphx**
This is an initial version of the security posture for securely running a PTFE instance.

### CODE REVIEW

File: content/source/docs/enterprise/private/security.html.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+PostgreSQL must be unsealed. For an external Vault, the unseal key(s) are a
+concern exclusively of that Vault cluster and are not in the path of PTFE.
+When using the interval Vault, the unseal key is encrypted and stored in a
+special table within PostgreSQL. The encryption key for the unseal key
+is the *Encryption Password* that is part of the PTFE settings. PTFE stores
+the *Encryption Password* on the local disk of the PTFE instance encrypted
+with a key instance encryption key.
+
+The ramifications of how the unseal key is stored when using the internal Vault
+is that a root user on the PTFE instance can extract the plaintext unseal key
```
Comment (koikonom) [inline content/source/docs/enterprise/private/security.html.md]: Should we add a blanket statement that root access on the instance will give people unlimited access to PTFE data (either by extracting passwords or by accessing the rails console) ?

Comment (evanphx) [reply content/source/docs/enterprise/private/security.html.md]: I added this!

**Comment by evanphx** [comment]
@armchairlinguist Yeah, the content is similar. The relationship is mostly to give a full picture of the security perspective without requiring the reader to jump around between pages to put the picture together.

**Comment by armchairlinguist** [comment]
@evanphx I was wondering if it might make more sense to just put the data security content here. Definitely I'd say we'd want to at least link to it and describe how it relates to this data security content.

**Comment by evanphx** [comment]
@armchairlinguist I'm open to doing that, sure.

**Comment by evanphx** [comment]
@armchairlinguist Thanks for pointing that out. It was confusing have a security and data security page so I merged them. All those details are in the same place now.

**Comment by nfagerlund** [comment]
This PR trailed off a bit and I'm not sure what the status is. I'm going to leave it open, since it looks like its still relevant, but it'll have to be reconstructed after the cloud / enterprise split.

**Comment by erindatkinson** [comment]
@evanphx is this PR still needed or is it okay to close?

**Comment by sudomateo** [comment]
Closing this pull request due to inactivity.

**Pull request closed without merging**